src/main/java/org/apache/fineract/api/OperationsDetailedApi.java

```diff
@@ -1,43 +1,34 @@
 package org.apache.fineract.api;
 
-
-import org.apache.fineract.operations.TransactionRequest;
-import org.apache.fineract.operations.TransactionRequestRepository;
-import org.apache.fineract.operations.TransactionRequestSpecs;
-import org.apache.fineract.operations.TransactionRequestState;
-import org.apache.fineract.operations.TransactionRequest_;
-import org.apache.fineract.operations.Transfer;
-import org.apache.fineract.operations.TransferRepository;
-import org.apache.fineract.operations.TransferSpecs;
-import org.apache.fineract.operations.TransferStatus;
-import org.apache.fineract.operations.Transfer_;
+import org.apache.fineract.data.ErrorCode;
+import org.apache.fineract.exception.WriteToCsvException;
+import org.apache.fineract.operations.*;
+import org.apache.fineract.utils.CsvUtility;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageRequest;
 import org.springframework.data.domain.Sort;
 import org.springframework.data.jpa.domain.Specifications;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
-
+import org.springframework.web.bind.annotation.*;
```

Comment:
lets avoid * import here as well

Reply:
should I remove * import from throughout the app?

Reply:
in this PR remove the places you have introduced it,   if you can do a sanity for the whole app, please send it as another PR